cs251tk/toolkit/args.py

```diff
@@ -165,4 +168,39 @@ def process_args():
     assignments = get_assignments_from_args(**args)
     stogit = compute_stogit_url(**args, _now=datetime.date.today())
 
+    print_args(args)
+    print_students(students)
+    print_assignments(assignments)
+    debug("stogit URL: " + stogit)
+
     return args, students, assignments, stogit
+
+
+def print_args(args):
+    debug("Command Line Arguments:")
+    for arg, value in args.items():
+        line = ""
+        line += arg + ":"
+        line += ' ' * (24 - len(arg))
+        line += str(value)
+        debug(line)
+
+
+def print_assignments(things):
+    debug("Assignments:")
+    print_grid(things)
+
+
+def print_students(students):
+    debug("Students:")
+    print_grid(students)
+
+
+def print_grid(items):
+    line = ""
+    for i, item in enumerate(items):
+        line += item
+        line += " " * (10 - len(item))
```

Comment:
Finally remembered what I was thinking of! `line += item.ljust(10)` will left-justify the text and pad to N chars with a space. It can replace both lines 202 and 203.

Reply:
That works